src/server/services/vergunningen-v2/vergunningen-route-handlers.ts

```diff
@@ -0,0 +1,94 @@
+import { Request, Response } from 'express';
+import { apiSuccessResult } from '../../../universal/helpers';
+import { BffEndpoints } from '../../config';
+import {
+  generateFullApiUrlBFF,
+  getAuth,
+  sendResponseContent,
+} from '../../helpers/app';
+import { fetchDecosZaakSource, fetchDecosZakenSource } from './decos-service';
+import { fetchVergunningDocumentV2, fetchVergunningV2 } from './vergunningen';
+import { DecosZaakSource } from './config-and-types';
+
+export async function fetchVergunningDetail(req: Request, res: Response) {
+  const authProfileAndToken = await getAuth(req);
+  const response = await fetchVergunningV2(
+    res.locals.requestID,
+    authProfileAndToken,
+    req.params.id
+  );
+
+  return res.send(response);
+}
+
+export async function fetchVergunningDocument(
+  req: Request<{ id: string }>,
+  res: Response
+) {
+  const authProfileAndToken = await getAuth(req);
+
+  const apiResponse = await fetchVergunningDocumentV2(
+    authProfileAndToken,
+    req.params.id
+  );
+
+  if (
+    'content' in apiResponse &&
+    'message' in apiResponse &&
+    apiResponse.status === 'ERROR'
+  ) {
+    return sendResponseContent(res, apiResponse);
+  }
+
+  apiResponse.data.pipe(res);
+}
+
+export async function fetchZakenSource(
```

Comment:
Ik zie dat die een aantal links meestuurt, als dat de 'source' of sources zijn is `fetchZaakSources` dan niet duidelijker? Ook omdat we Source weleens als bronsysteem gebruiken. Misschien is dit ook wel een beetje een nitpick

Reply:
Ik snap niet helemaal wat je bedoelt, er worden geen links meegestuurd?